main.cpp

```diff
@@ -4256,11 +4256,13 @@ bool load_guts(picoboot::connection con, iostream_memory_access &file_access) {
             }
         }
     }
-    for (auto mem_range : ranges) {
-        enum memory_type type = get_memory_type(mem_range.from, model);
-        // new scope for progress bar
-        {
-            progress_bar bar("Loading into " + memory_names[type] + ": ");
+    size_t total_bytes = 0;
+    for (auto r : ranges) { total_bytes += r.to - r.from; }
+    {
```

Comment:
It's probably worth keeping the `// new scope for progress bar` comment, oherwise it might look like the `{` on the second line is associated with the `for` on the previous line :wink: 
Or alternatively, as there's now not multiple progress bars, perhaps this additional scope is no longer necessary? :thinking: